README.md

```diff
@@ -2,79 +2,80 @@
 
 # AICA Package Template
 
-A software development kit for creating custom components for AICA applications.
+A software development kit for creating custom components and controllers for AICA applications.
```

Comment:
I recommend we introduce what a package is and the difference between a mono-package and collection package, since it's referenced later. Keeping it brief so as not to duplicate too much of the formal docs pages, but still have enough to stand alone:

```suggestion
A software development kit for creating custom component and controller extensions for AICA applications.

Custom extensions are built and distributed in ROS 2 compatible packages with additional metadata for
the AICA System. When an AICA package contains multiple ROS 2 packages, it is referred to as a _collection_.
```